### ISSUE TITLE
Not restarting

### ISSUE THREAD

**Issue by ChompinPacman** [BTD6Scripts/BTD6-Auto-Grinder#1]
Everything works up to the restart, when the lvl ends with "victory" it doesn't restart. I'm trying to learn more about PixelSearch, I assume that's the issue.

**Comment by Fullius** [comment]
Same thing's happening to me.

**Comment by Jigsaw1k** [comment]
The script does not place the ninja monkey for it, it goes to place it but it is off the mpa in the lane by 5cm. is it wokring for you guys ?

**Comment by Fullius** [comment]
It places everything, but it doesn't restart. Maybe you could try opening the file in like VS Code and change the pixels where it places it a little, like 5-10, then try running it again.

**Comment by Jigsaw1k** [comment]
Yea not sure, places everything but the ninja monkey and for some reason it spams spacebar every 2-3 seconds, kind of a bummer there are no other working scripts, even this one only farms about 90kxp per 10 mins

**Comment by Fullius** [comment]
What's your computers aspect ratio if you don't mind me asking?

**Comment by Jigsaw1k** [comment]
i play one 3840x1440 but i changed my btd6 res to fullscreen 1920x1080

**Comment by Fullius** [comment]
Gimme a sec, I'll give the program another go.

**Comment by Jigsaw1k** [comment]
yea, i mean if the game restart isnt even working i dont really care about the placement i will just not use the program

**Comment by Fullius** [comment]
It places everything correctly. I might try and remake this because tbh, I'm not bothered to grind on my own and I want to be able to get monkey money since my skill level is below 1. I'll let you know how it goes. The program did say something about having a 16:9 aspect ratio - yours is 21:9. That might be the entire problem.

Edit: Fixed 19:6 to 16:9

**Comment by Jigsaw1k** [comment]
I have way more monkey money then needed lol just want the levels. Heres my discord if ya wanna chat there 

Clownz#2477

**Comment by Fullius** [comment]
Ah okay. Yeah, I'll DM you when it's done to see how it works on yours, the monitor I have and the laptop I'm currently using (All different aspect ratios).

**Comment by Jigsaw1k** [comment]
ill switch ratios after the game im on is done and test it out

**Comment by Jigsaw1k** [comment]
got it working, now to see about the restart problem

**Comment by Fullius** [comment]
Yeah the restart problem looks like it's caused because it isn't actually taking any exact spots - it's trying to find them itself.

**Comment by Jigsaw1k** [comment]
yea excatly, it waits for the game to end and then nothing happens other then mouse clicks in the center of the screen, too bad it doesnt work properly anymore

**Comment by Fullius** [comment]
I'll do some tinkering with this file to try and fix it - and hope it doesn't break anything or get my account flagged.

**Comment by ugsusu** [comment]
Same here, it works up to the victory screen. I think the victory screen might've changed and it's looking in the wrong place. I tried adjusting it in the code but I don't quite understand how these parameters work `PixelSearch, , , 0.9540 * A_ScreenWidth, 0.9213 * A_ScreenHeight, 0.9580 * A_ScreenWidth, 0.9250 * A_ScreenHeight` I tried adjusting it according to the AHK guide with the monitor coordinates but now the cursor just jumps to the top left corner of the screen `PixelSearch, , , 1117, 164, 1306, 301, 0xDB2100, 2, Fast RGB	; look for the orange band to the right of "VICTORY" to restart faster`

**Comment by mertisthemerty** [comment]
> I'll do some tinkering with this file to try and fix it - and hope it doesn't break anything or get my account flagged.

did you end up getting it working lol, works until the victory screen
